packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -203,24 +212,24 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
       this._async.setTimeout(() => this._scrollIntoView(), 0);
     }
 
-    // If we are open or we are focused but are not the activeElement,
-    // set focus on the input
-    if (isOpen || (focused && document.activeElement !== this._comboBox.inputElement)) {
+    // If we are open or we are just closed, should focusAfter close, are focused but
```

Comment:
nit: replace should focusAfter close with shouldFocusAfterClose is set

Reply:
done